locale/en_US/manager.po

```diff
@@ -266,6 +266,21 @@ msgstr "Items can only be submitted by Editors and Section Editors."
 msgid "manager.sections.editors"
 msgstr "Section editors"
 
+msgid "manager.sections.inactive"
+msgstr "Inactive"
+
+msgid "manager.sections.form.deactivateSection"
+msgstr "Deactivate section (disables new submissions)"
+
+msgid "manager.sections.confirmDeactivateSection"
+msgstr "Are you sure you wish to deactivate this section?"
+
+msgid "manager.sections.confirmActivateSection"
+msgstr "Are you sure you wish to activate this section?"
```

Comment:
Any locale keys that don't use OJS-specific terminology, like "journal" or "article", can be moved to pkp-lib. That way they can be re-used in OMP/OPS.